client/lcd/lcd_test.go

```diff
@@ -59,9 +59,11 @@ func TestKeys(t *testing.T) {
 	require.Equal(t, http.StatusOK, res.StatusCode, body)
 	var resp keys.NewKeyResponse
 	err = wire.Cdc.UnmarshalJSON([]byte(body), &resp)
-	require.Nil(t, err)
-	addr2 := resp.Address
-	assert.Len(t, addr2, 40, "Returned address has wrong format", addr2)
+	require.Nil(t, err, body)
+
+	addr2Bech32 := resp.Address
+	_, err = sdk.GetAccAddressBech32(addr2Bech32)
+	assert.NoError(t, err, "Did not return a correct bech32 address")
```

Comment:
Don't wanna bit nitpicky on nomenclature here, but do you think `Failed to return a correct bech32 address` would sound better?

Reply:
Hahahah I don't mind. Feel free to change this on my branch.